package.json

```diff
@@ -31,13 +32,14 @@
 		"check"
 	],
 	"dependencies": {
-		"p-cancelable": "^0.4.1"
+		"p-cancelable": "^1.1.0"
 	},
 	"devDependencies": {
-		"ava": "*",
-		"delay": "^2.0.0",
-		"jsdom": "^11.10.0",
-		"xo": "*"
+		"ava": "^1.3.1",
+		"delay": "^4.1.0",
+		"jsdom": "^14.0.0",
+		"tsd-check": "^0.3.0",
+		"xo": "^0.24.0"
```

Comment:
Unwanted change? @sindresorhus 

```diff
-		"xo": "^0.24.0"
+  		"xo": "*"
```

Reply:
No, I started setting versions for everything a while back. Got too many problems with `*`.